backend/lucene/src/main/java/org/hibernate/search/backend/lucene/analysis/impl/TokenizerChain.java

```diff
@@ -54,6 +54,15 @@ protected TokenStreamComponents createComponents(String fieldName) {
 		return new TokenStreamComponents( tk, ts );
 	}
 
+	@Override
+	protected TokenStream normalize(String fieldName, TokenStream in) {
```

Comment:
I was surprised to see only token filters are mentioned here. It seems `CustomAnalyzer` also overrides another method to make sure char filters are applied: `org.apache.lucene.analysis.custom.CustomAnalyzer#initReaderForNormalization`. I think you should also copy this method. Maybe just below `initReader`?